.travis.yml

```diff
@@ -27,20 +51,35 @@ cache:
     - $HOME/.android/build-cache
 
 stages:
-  - test
+  - name: test
   - name: deploy
     if: tag IS present
 
+before_script:
+  - echo y | sdkmanager "system-images;$TEST_PLATFORM;google_apis;x86" > /dev/null
+  - echo no | avdmanager create avd -f -n "$TEST_PLATFORM" -k "system-images;$TEST_PLATFORM;google_apis;x86" -c 500M
+  - emulator -avd "$TEST_PLATFORM" -memory 4096 -no-audio -no-snapshot -no-window -camera-back none -camera-front none &
```

Comment:
TODO: will adding `-skin` help control the resource consumption.

Reply:
https://developer.android.com/studio/run/emulator-commandline#deprecated `-skin` is deprecated.